packages/workflows-engine-inmemory/src/services/workflows-module.ts

```diff
@@ -0,0 +1,199 @@
+import {
```

Comment:
suggestion: DTOs are missing for some method arguments. Would make it a bit cleaner